<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow1</class>
 <widget class="QMainWindow" name="MainWindow1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1370</width>
    <height>1035</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="toggle_button" name="pushButton_grid">
            <property name="text">
             <string>网格</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="toggle_button" name="pushButton_timeline">
            <property name="text">
             <string>时间线</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="toggle_button" name="pushButton_playlist">
            <property name="text">
             <string> 播放列表</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="toggle_button" name="play_button">
            <property name="text">
             <string> 播放</string>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="toggle_button" name="output_button">
            <property name="text">
             <string>  输出到激光</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget_2">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="page_13">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <property name="minimumSize">
              <size>
               <width>992</width>
               <height>583</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="Scene_Tool_Box" name="m_scenes_book">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="page_14">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>98</width>
                   <height>28</height>
                  </rect>
                 </property>
                 <attribute name="label">
                  <string>Page 1</string>
                 </attribute>
                </widget>
                <widget class="QWidget" name="page_15">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>98</width>
                   <height>28</height>
                  </rect>
                 </property>
                 <attribute name="label">
                  <string>Page 2</string>
                 </attribute>
                </widget>
               </widget>
              </item>
              <item>
               <widget class="Scene_Stack" name="m_scenes_stack">
                <property name="minimumSize">
                 <size>
                  <width>700</width>
                  <height>521</height>
                 </size>
                </property>
                <property name="currentIndex">
                 <number>1</number>
                </property>
                <widget class="QWidget" name="page_16"/>
                <widget class="QWidget" name="page_17">
                 <layout class="QGridLayout" name="gridLayout_3">
                  <item row="0" column="6">
                   <widget class="QGraphicsView" name="graphicsView_87">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="5">
                   <widget class="QGraphicsView" name="graphicsView_94">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QGraphicsView" name="graphicsView_90">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="4">
                   <widget class="QGraphicsView" name="graphicsView_109">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="5">
                   <widget class="QGraphicsView" name="graphicsView_118">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="2">
                   <widget class="QGraphicsView" name="graphicsView_115">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QGraphicsView" name="graphicsView_105">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="7">
                   <widget class="QGraphicsView" name="graphicsView_88">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="3">
                   <widget class="QGraphicsView" name="graphicsView_108">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QGraphicsView" name="graphicsView_98">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="7">
                   <widget class="QGraphicsView" name="graphicsView_112">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QGraphicsView" name="graphicsView_83">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="3">
                   <widget class="QGraphicsView" name="graphicsView_100">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="7">
                   <widget class="QGraphicsView" name="graphicsView_96">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="3">
                   <widget class="QGraphicsView" name="graphicsView_92">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QGraphicsView" name="graphicsView_97">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="4">
                   <widget class="QGraphicsView" name="graphicsView_93">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QGraphicsView" name="graphicsView_89">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="6">
                   <widget class="QGraphicsView" name="graphicsView_103">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="7">
                   <widget class="QGraphicsView" name="graphicsView_104">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="4">
                   <widget class="QGraphicsView" name="graphicsView_117">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="2">
                   <widget class="QGraphicsView" name="graphicsView_107">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="7">
                   <widget class="QGraphicsView" name="graphicsView_120">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="5">
                   <widget class="QGraphicsView" name="graphicsView_110">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="5">
                   <widget class="QGraphicsView" name="graphicsView_86">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QGraphicsView" name="graphicsView_81">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="6">
                   <widget class="QGraphicsView" name="graphicsView_95">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="5">
                   <widget class="QGraphicsView" name="graphicsView_102">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="6">
                   <widget class="QGraphicsView" name="graphicsView_119">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="3">
                   <widget class="QGraphicsView" name="graphicsView_116">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="2">
                   <widget class="QGraphicsView" name="graphicsView_91">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QGraphicsView" name="graphicsView_106">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="6">
                   <widget class="QGraphicsView" name="graphicsView_111">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="3">
                   <widget class="QGraphicsView" name="graphicsView_84">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="0">
                   <widget class="QGraphicsView" name="graphicsView_113">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="2">
                   <widget class="QGraphicsView" name="graphicsView_99">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="1">
                   <widget class="QGraphicsView" name="graphicsView_114">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="4">
                   <widget class="QGraphicsView" name="graphicsView_101">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="4">
                   <widget class="QGraphicsView" name="graphicsView_85">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QGraphicsView" name="graphicsView_82">
                    <property name="minimumSize">
                     <size>
                      <width>105</width>
                      <height>105</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_18">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="project_panel" name="project_panel_view"/>
              </item>
              <item>
               <widget class="QWidget" name="widget_7" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QPushButton" name="pushButton_2">
                   <property name="text">
                    <string>PushButton</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>713</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_3">
                   <property name="text">
                    <string>PushButton</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton">
                   <property name="text">
                    <string>PushButton</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="quick_tabWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="Scene_Tool_Box" name="quick_scenes_book">
             <property name="maximumSize">
              <size>
               <width>68</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="page_3">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>68</width>
                <height>205</height>
               </rect>
              </property>
              <attribute name="label">
               <string>Page 1</string>
              </attribute>
             </widget>
             <widget class="QWidget" name="page_4">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>68</width>
                <height>205</height>
               </rect>
              </property>
              <attribute name="label">
               <string>Page 2</string>
              </attribute>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="Scene_Stack" name="quick_scenes_stack">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="page"/>
             <widget class="QWidget" name="page_2"/>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_6" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="output_panel" name="display_view">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>300</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>300</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1370</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="hardware"/>
    <addaction name="laser_setting"/>
    <addaction name="projection_zones"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="new_workspace"/>
    <addaction name="open_workspace"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="hardware">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>激光灯与控制器硬件</string>
   </property>
  </action>
  <action name="laser_setting">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>激光灯设置</string>
   </property>
  </action>
  <action name="projection_zones">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>投影区域</string>
   </property>
  </action>
  <action name="new_workspace">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>新建工作区</string>
   </property>
  </action>
  <action name="open_workspace">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>打开工作区</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Scene_Tool_Box</class>
   <extends>QToolBox</extends>
   <header>scene_tool_box.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Scene_Stack</class>
   <extends>QStackedWidget</extends>
   <header>scene_tool_box.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>output_panel</class>
   <extends>QGraphicsView</extends>
   <header>output_panel.h</header>
  </customwidget>
  <customwidget>
   <class>project_panel</class>
   <extends>QGraphicsView</extends>
   <header location="global">project_panel.h</header>
  </customwidget>
  <customwidget>
   <class>toggle_button</class>
   <extends>QPushButton</extends>
   <header>toggle_button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
